ui/00_message/ui/system_log.xml

```diff
@@ -13,19 +13,19 @@
   </message>
   <message>
     <key>syslog_tail_on</key>
-    <text xml:space="preserve">&lt;NAME TARGET&gt;に追従オートランしました</text>
+    <text xml:space="preserve">&lt;NAME TARGET&gt; auto run has been initiated.</text>
```

Comment:
This line means, "You began following &lt;NAME TARGET&gt; " it even says it in the key "tail", if you try to translate the first two kanjis, they translate in "following"